### PR TITLE
DomainTools Iris Investigate and Enrich Custom Connectors

### PR DESCRIPTION
**Pull request by RamboV**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to a high volume of connector submissions. It may take 1-2 weeks for us to provide comments for your connector.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,1145 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DomainTools Iris Enrich",
+    "description": "Receive immediate context to indicators to bolster triage, response, and defense.",
```
Comment (sriyen-msft) [inline certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json:5]: @RamboV Can you please provide some more details of your connector which should describe concisely the main purpose and value offered by your connector?

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1145 @@
...
+            },
+            "message": {
+              "type": "string",
+              "description": "message",
+              "title": "Message"
+            },
+            "results_count": {
+              "type": "integer",
+              "format": "int32",
+              "description": "results_count",
```
Comment (sriyen-msft) [inline certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json:190]: @RamboV All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,1145 @@
...
+                      "value": {
+                        "type": "string",
+                        "description": "Adsense Value",
+                        "title": "Adsense Value"
+                      }
+                    },
+                    "description": "adsense"
+                  },
+                  "alexa": {
+                    "additionalProperties": {},
```
Comment (sriyen-msft) [inline certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json:220]: @RamboV Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json (hunk 1 above)
Comment (RamboV) [reply certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json]: Hi @sriyen-msft , Modified the descriptions

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json (hunk 2 above)
Comment (RamboV) [reply certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json]: Hi @sriyen-msft , Modified the parameter descriptions

File: certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json (hunk 3 above)
Comment (RamboV) [reply certified-connectors/DomainToolsIrisEnrich/apiDefinition.swagger.json]: Hi @sriyen-msft , yes this is left intentionally for a few response properties, as for few of the response properties datatypes are dynamic in nature.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**